### PR TITLE
Add Terratest unit-test files

### PR DESCRIPTION
**Pull request by zuriguardiola**

### CODE REVIEW

File: test/unit-test/main.tf [hunk 1]
```diff
@@ -0,0 +1,229 @@
+module "ecs" {
+
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-ecs?ref=61ed2af4ba963647931ed0359939787a0f0d74f7"
```
Comment (dms1981) [inline test/unit-test/main.tf]: shouldn't this refer to the local root directory, so the code being tested is always to most recent version?

**Review by dms1981 — DISMISSED** [review]

**Comment by dms1981** [comment]
Also I think you're going to want to address the SCA failures; they look pretty straightforward though.

File: test/unit-test/main.tf (hunk 1 above)
Comment (zuriguardiola) [reply test/unit-test/main.tf]: Will update with a tagged version once these changes are merged

**Comment by zuriguardiola** [comment]
> Also I think you're going to want to address the SCA failures; they look pretty straightforward though.

SCA failures have been fixed

**Review by davidkelliott — APPROVED** [review]

**Pull request merged**